packages/shared/src/hooks/useFeedLayout.ts

```diff
@@ -75,17 +76,19 @@ const getFeedPageLayoutComponent = ({
   shouldUseListFeedLayoutOnMobileTablet,
   shouldUseCommentFeedLayout,
   shouldUseListMode,
+  isSearchResultsUpgrade,
 }: Pick<
   UseFeedLayoutReturn,
   'shouldUseCommentFeedLayout' | 'shouldUseListMode'
 > & {
   shouldUseListFeedLayoutOnMobileTablet: boolean;
+  isSearchResultsUpgrade: boolean;
 }): UseFeedLayoutReturn['FeedPageLayoutComponent'] => {
   if (shouldUseCommentFeedLayout) {
     return CommentFeedPage;
   }
 
-  if (shouldUseListMode) {
+  if (shouldUseListMode && !isSearchResultsUpgrade) {
```

Comment:
Does this mean even if the user is in list mode but enrolled in the experiment, they will see the grid layout?

Reply:
`isSearchResultsUpgrade` is true only in the search page feed, only scree width > laptop and if you enrolled the experiment. None of the other feeds should be changed by this addition